CHANGELOG.md

```diff
@@ -35,6 +35,7 @@ Table of Contents
 ### New Features
 
 * The `ProjectLayout` may now retrieve the test source root for the current project in order to allow for test healing to be applied. This is required for custom `ProjectLayouts` to be implemented, in order for this feature to work. For more information, please refer to the documentation.
+* Use `import: $reference` to import another filter into an filter or an ignore file. Ignoring all positioning changes is now easily possible with `import: positioning.filter`. Whole cascades and hierarchies of filters can now be created this way. 
```

Comment:
Use _a filter_.

Reply:
Sorry ... I don't see it. Where is the typo?

Reply:
Damn, I was too quick with approving :D. It would have been at:

> to import another filter into **an filter** or an ignore file

Reply:
Sorry, I still don't see it. Isn't this already **an filter**?

Reply:
```suggestion
* Use `import: $reference` to import another filter into a filter or an ignore file. Ignoring all positioning changes is now easily possible with `import: positioning.filter`. Whole cascades and hierarchies of filters can now be created this way. 
```

Reply:
Now I see it. Thought the typo was in filter ... Thanks.